hermes-console/static/partials/group.html

```diff
@@ -7,17 +7,17 @@
     </ol>
 
     <div class="pull-right">
-        <button class="btn btn-primary" ng-click="edit()">Edit</button>
-        <button class="btn btn-danger" ng-click="remove()">Remove</button>
+        <button class="btn btn-primary" ng-show="admin" ng-click="edit()">Edit</button>
+        <button class="btn btn-danger" ng-show="admin" ng-click="remove()">Remove</button>
     </div>
 
     <h1><small>Group:</small> {{group.groupName}}</h1>
 
-    <div class="row">
+    <div class="row container">
         <div class="list-group">
             <div class="list-group-item">
                 <div class="pull-right">
-                    <button class="btn btn-primary" ng-click="addTopic()"><span class="glyphicon glyphicon-plus"></span></button>
+                    <button class="btn btn-primary" ng-disabled="disabled" ng-click="addTopic()"><span class="glyphicon glyphicon-plus"></span></button>
```

Comment:
I think this button should be enabled when not logged in, we have this feature of `root password` that allows creating topics by anonymous users.